trunner/config.py

```diff
@@ -30,7 +30,7 @@ def resolve_phrtos_dir() -> Path:
 PYEXPECT_TIMEOUT = 8
 
 # Available targets for test runner.
-ALL_TARGETS = ['ia32-generic', 'host-pc', 'armv7m7-imxrt106x']
+ALL_TARGETS = ['ia32-generic', 'host-pc', 'armv7m7-imxrt106x', 'armv7m4-stm32l4']
```

Comment:
maybe we should remove `armv7m4-stm32l4` from `DEFAULT_TARGETS` if not all tests are runnable right away (I assume this will change in the near future)

Reply:
Good point. Removed